README.md

```diff
@@ -53,6 +53,57 @@ npm install --save-dev mock-inspect
 yarn add -D mock-inspect
 ```
 
+# Setting up your test suite
+
+Our mocking solution has to be set up and torn down accordingly after tests. The method `cleanUpNetworkRequestMocking` should run before **each** of your tests, `setUpNetworkRequestMocking()` **once** before all of your tests, and `tearDownNetworkRequestMocking` **once** after all of your tests. How setup steps have to be invoked depends on your test runner. With jest, we would create a [setupTestFrameworkFile](https://jestjs.io/docs/en/23.x/configuration#setuptestframeworkscriptfile-string) which would register the following hooks:
+
+```js
+const {
+    cleanUpNetworkRequestMocking,
+    setUpNetworkRequestMocking,
+    tearDownNetworkRequestMocking
+} = require("mock-inspect")
+
+beforeEach(() => {
+    cleanUpNetworkRequestMocking()
+})
+
+beforeAll(() => {
+    setUpNetworkRequestMocking()
+})
+
+afterAll(() => {
+    tearDownNetworkRequestMocking()
+})
+```
+
+## Persistent auto-generated graphQL responses in the setup phase
+
+For graphQL APIs, we expose functionality in the setup phase to automatically generate graphQL responses from a schema throughout your entire test suite. (These can be overridden if necessary though). The below example sets up the test suite in such a way so that any graphQL request going against https://thisdomaindoesnotexist.com/graphql will be evaluated against the provided graphQLSchema and an automatic response will be generated. [Refer to the type definitions](https://github.com/trayio/mock-inspect/blob/main/src/types/GraphQLAutoMockStartupOptions.ts) for all available options.
+
+```js
+beforeAll(() => {
+    setUpNetworkRequestMocking({
+        persistentAutoGeneratedGraphQLMocks: [
+            {
+                requestPattern: "https://thisdomaindoesnotexist.com/graphql",
+                graphQLSchema: schemaString,
```

Comment:
What's this `schemaString` does it need to be imported?

Reply:
I can put the same comment here like I already did on the other section about auto mocking, thanks.